pkg/proxy/userspace/proxysocket.go

```diff
@@ -95,18 +95,18 @@ func TryConnectEndpoints(service proxy.ServicePortName, srcAddr net.Addr, protoc
 	for _, dialTimeout := range EndpointDialTimeouts {
 		endpoint, err := loadBalancer.NextEndpoint(service, srcAddr, sessionAffinityReset)
 		if err != nil {
-			klog.Errorf("Couldn't find an endpoint for %s: %v", service, err)
+			klog.ErrorS(err, "Couldn't find an endpoint for service", "service", service)
 			return nil, err
 		}
-		klog.V(3).Infof("Mapped service %q to endpoint %s", service, endpoint)
+		klog.V(3).InfoS("Mapped service to endpoint", "serivce", service, "endpoint", endpoint)
```

Comment:
misspelled "service'

Reply:
Ahh, my bad, thanks for pointing out, fixed